### PR TITLE
Adds new power generation method.  Throwing a coin into supermatter.

### PR DESCRIPTION
**Pull request by lordpidey**
Low cost, high speed, high power generation, 10% chance of throwing you into crit.

### CODE REVIEW

File: src/state/jobSingleAction.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ export default {
 			if (action.failure) {
 				if (Math.random() < action.failure.chance) {
 					// At least try again
-					EventBus.$emit("toast", { text: "Apprehended!" });
+					EventBus.$emit("toast", { text: action.failure.message });
```
Comment (TBartl) [inline src/state/jobSingleAction.js:132]: Nice

File: src/data/engineering.js [hunk 2]
```diff
@@ -254,6 +254,26 @@ const ILL_ADVISED_ACTIONS = {
 			power: 11
 		}
 	},
+	engSuperMatter: {
+		time: 5,
+		actionName: "THROW COIN AT",
+		name: "Supermatter Crystal",
+		failure: {
+			chance : 0.1,
+			damage: 1000,
```
Comment (TBartl) [inline src/data/engineering.js:263]: You might want to bump this up to 9999 or something, unless your intention is that players should be able to survive it by gearing up (I think the health cap is >1200 right now or something)

File: src/data/engineering.js [hunk 3]
```diff
@@ -254,6 +254,26 @@ const ILL_ADVISED_ACTIONS = {
...
+		actionName: "THROW COIN AT",
+		name: "Supermatter Crystal",
+		failure: {
+			chance : 0.1,
+			damage: 1000,
+			message: "Dusted"
+		},
+		items: {
+			id: "power",
+			count: 100
```
Comment (TBartl) [inline src/data/engineering.js:268]: I'm not sure I'm a super big fan of this being the highest power/s in the game. You could realistically go nude and actively play to get much better power gen than bananium, oil slimes, or any other good power generation method that relies on other skills.

I'm OK with active play rewarding _slightly_ higher power/s, but right now this is giving ~18 power / s, while oil slimes (the current best method in the game) give ~3 power/s

So I'd say either
- Bump up the time significantly:
- Lower the power gain
- Put some hefty -xp on this (making it a better, but more risky singulo)
- Make it a lot of coins instead

Comment (lordpidey) [reply src/data/engineering.js:268]: You bring up a good point.

I will do a little math tomorrow to estimate which of these I should do.

**Pull request merged**